crates/services/p2p/src/gossipsub/config.rs

```diff
@@ -25,9 +25,57 @@ use sha2::{
 };
 use std::time::Duration;
 
-use super::topics::GossipTopic;
+use super::topics::{
+    GossipTopic,
+    CON_VOTE_GOSSIP_TOPIC,
+    NEW_BLOCK_GOSSIP_TOPIC,
+    NEW_TX_GOSSIP_TOPIC,
+};
+
+// The number of slots in each epoch.
+const SLOTS_PER_EPOCH: u64 = 32;
+
+// The duration of each slot in seconds. This is the amount of time allotted for each opportunity to create a block.
```

Comment:
What is the "block" here?

Reply:
SLOT represents a unit of time, in this case it represents time between two produced fuel blocks